Sources/ArcGISToolkit/Components/Offline/Preplanned/PreplannedMapModel.swift

```diff
@@ -215,7 +215,13 @@ class PreplannedMapModel: ObservableObject, Identifiable {
         case .success:
             status = .downloaded
         case .failure(let error):
-            status = .downloadFailure(error)
+            if error is CancellationError {
```

Comment:
This is modeled after https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/main/Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandMapModel.swift#L266